tests/test_mbfxml2ex.py

```diff
@@ -221,7 +221,7 @@ def test_write_tree_with_markers(self):
             os.remove(ex_file)
 
         write_ex(ex_file, neurolucida_data)
-        self.assertTrue(os.path.exists(ex_file))
```

Comment:
Probably want to keep this line as the next assert will assume it exists.

Reply:
Restored.